Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java

```diff
@@ -87,45 +88,52 @@ public void initialize(URL url, ResourceBundle rb) {
     }
 
     /**
-     * Method that display the current settings of the room on the session view.
+     * Method that displays a summary of the game on the GameResults scene.
      */
     public void setGameInfo() {
         Room room = currentSession.getRoom();
         Game game = currentSession.getGame();
 
-        for(Player p : room.getTeamBlue().getPlayers()) {
+        for (Player p : room.getTeamBlue().getPlayers()) {
             int goals = 0;
+            for (Event e : game.getEvents()) {
+                if (e.getType() == EventType.GOAL_BLUE && e.getPlayer().equals(p)) {
+                    goals++;
+                }
+            }
+
             ResultTableItem r = new ResultTableItem(p.getUsername(), goals);
             tblBlueGoalsList.getItems().add(r);
         }
-      
-        for(Player p : room.getTeamRed().getPlayers()) {
+
+        for (Player p : room.getTeamRed().getPlayers()) {

```

Comment:
Ook is dit hele blok gedupliceerd. Gelieve een methode maken die een Team parameter neemt zodat je daar Team blue en Red in kan meegeven (2x aanroepen).